### PR TITLE
Trying to get property 'type' of non-object issue.

### PR DESCRIPTION
**Pull request by mhmelshaaer**

### CODE REVIEW

File: resources/views/tools/bread/relationship-partial.blade.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@php $relationshipDetails = $relationship['details']; @endphp
+@php $relationshipDetails = json_decode($relationship['details']); @endphp
```
Comment (emptynick) [inline resources/views/tools/bread/relationship-partial.blade.php:1]: This attribute is already casted here. I don't see a reason why we should `json_decode`
 it again
https://github.com/the-control-group/voyager/blob/6c9d6289bedd7e8db22b020aabd5789aa20d7dbc/src/Models/DataType.php#L244

**Review by emptynick — CHANGES_REQUESTED** [review]

File: resources/views/tools/bread/relationship-partial.blade.php (hunk 1 above)
Comment (mhmelshaaer) [reply resources/views/tools/bread/relationship-partial.blade.php:1]: When browsing or editing User BREAD after a fresh voyager installation, for some reason it throws "try to get property of non-object" error, on editing line 54 in relationship-partial.blade.php "$relationshipDetails->type" throws an error and on browsing line 16 in BreadRelationshipParser.php "$row->details->type"  throws an error. This issue happens a lot, I tried solving it by editing those lines and it worked, and now, magically, after a while, it throws an error for those edits and works just fine after undoing the edits.

Comment (emptynick) [reply resources/views/tools/bread/relationship-partial.blade.php:1]: See https://github.com/the-control-group/voyager/issues/3871
Turns out there was invalid (not installed by default) json in his database

Comment (MouhammedElshaaer) [reply resources/views/tools/bread/relationship-partial.blade.php]: yes, I've checked that, but it was already set to 1. Maybe I'll try another fresh voyager installation and will see if this happens.

Comment (mhmelshaaer) [reply resources/views/tools/bread/relationship-partial.blade.php:1]: Oh, I see. Thanks, I will close this after you see this comment.

**Pull request closed without merging**